<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnedriveDialog</class>
 <widget class="QDialog" name="OnedriveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RememberKey -- OneDrive</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>9</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabSighIn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Sign In</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Input Authorization Code from Web Browser:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeInput">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="goButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSelect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Select File</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>41</y>
       <width>281</width>
       <height>201</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="addFolderButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+ Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addFileButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+ File</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select a file to be sync with:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goupButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Go Up</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabProgress">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Progress</string>
    </attribute>
    <widget class="QLabel" name="downloadLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Download Progress</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="uploadLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload Progress</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>codeInput</tabstop>
  <tabstop>goButton</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>goupButton</tabstop>
  <tabstop>addFolderButton</tabstop>
  <tabstop>addFileButton</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
